tests/app/data/test_schema_validation.py

```diff
@@ -0,0 +1,47 @@
+from app.schema_loader import schema_loader
+import json
+import jsonschema
+import logging
+import unittest
+import os
+
+from app import settings
+
+from jsonschema import validate
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestSchemaValidation(unittest.TestCase):
+
+    def test_schema(self):
+
+      errors = []
+
+      for file in schema_loader.available_schemas():
```

Comment:
It might be better to just read the json files directly from the data directory here - I got an error trying to read an s3 schema from the data dir:
`FileNotFoundError: [Errno 2] No such file or directory: 'app/data/1.json'`